docs/clusterdefinition.md

```diff
@@ -10,12 +10,21 @@ Here are the cluster definitions for apiVersion "vlabs"
 |---|---|---|
 |apiVersion|yes|The version of the template.  For "vlabs" the value is "vlabs".|
 
+### location
+`location` describe the deployment target of the cluster. Current available: 
+```    
+ "australiaeast", "australiasoutheast", "brazilsouth", "canadacentral", "canadaeast", "centralindia", "centralus", "eastasia", "eastus", "eastus2", "japaneast", "japanwest", "koreacentral", "koreasouth", "northcentralus", "northeurope", "southcentralus", "southeastasia", "southindia", "uksouth", "ukwest", "westcentralus", "westeurope", "westindia", "westus", "westus2", "chinaeast", "chinanorth"
+```
+
+This value can be ignored by default if the deployment is go to public azure. However, if the deployment is go to non public azure such `azure china`, the correct value must be specified. ACS engine will based on this value to determine the download URLs of the required installation binaries such Docker, DC/OS, Kubernetes Docker images. Because different location may have different policy such as China Great Firewall which blocks Azure CDN and GCR.

```

Comment:
does not need to be specified if the deployment location is a Public Azure region. It would be useful if Public Azure regions are specified separately or even better would be refer to Azure's official region list then this list is not be out of date if new regions are added.